source/configuration_manual/config_file/config_variables.rst

```diff
@@ -358,7 +358,7 @@ Authentication variables:
 |          |                     |                                                               |
 |          |                     | .. versionadded:: v2.2.29                                     |
 +----------+---------------------+---------------------------------------------------------------+
-|          | forward_<name>      | Used by proxies to pass on extra fields to the next hop, see  |
+|          | passdb:forward_<name> | Used by proxies to pass on extra fields to the next hop, see  |
```

Comment:
Right. But I think the whole table needs to be reformatted, since the field doesn't fit into the cell..

Reply:
sorry, was unsure about how the formatting of things changed (was hoping it would still render it to html nicely on it's own) - want me to update all rows, or do you want to do it?

Reply:
Easier if you could just do it. :)

Reply:
done!